Fix errant log message

I was disconnected from the network and fired up Eclipse to see a panicky error from `ManagedCloudSdkStartup`
```
May 18, 2018 8:51:30 AM com.google.cloud.tools.eclipse.ui.status.PollingStatusServiceImpl refreshStatus
WARNING: Cannot connect to GCP: java.net.NoRouteToHostException: Can't assign requested address (Address not available)
May 18, 2018 8:51:30 AM com.google.cloud.tools.eclipse.sdk.ui.ManagedCloudSdkStartup$1$1 run
SEVERE: Corrupt Cloud SDK installation?
com.google.cloud.tools.managedcloudsdk.ManagedSdkVerificationException: com.google.cloud.tools.managedcloudsdk.command.CommandExitException: Process failed with exit code: 1
	at com.google.cloud.tools.managedcloudsdk.ManagedCloudSdk.isUpToDate(ManagedCloudSdk.java:157)
	at com.google.cloud.tools.eclipse.sdk.ui.ManagedCloudSdkStartup.checkInstallation(ManagedCloudSdkStartup.java:114)
	at com.google.cloud.tools.eclipse.sdk.ui.ManagedCloudSdkStartup$1$1.run(ManagedCloudSdkStartup.java:89)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: com.google.cloud.tools.managedcloudsdk.command.CommandExitException: Process failed with exit code: 1
	at com.google.cloud.tools.managedcloudsdk.command.CommandCaller.call(CommandCaller.java:68)
	at com.google.cloud.tools.managedcloudsdk.ManagedCloudSdk.isUpToDate(ManagedCloudSdk.java:149)
	… 3 more
```

--- a/plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/ManagedCloudSdkStartup.java
+++ b/plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/ManagedCloudSdkStartup.java
@@ -72,35 +72,36 @@ public class ManagedCloudSdkStartup implements IStartup {
       return;
     }
 
+    Job checkInstallationJob = new Job("Check Google Cloud SDK") {
+      @Override
+      protected IStatus run(IProgressMonitor monitor) {
+        try {
+          CloudSdkManager sdkManager = CloudSdkManager.getInstance();
+          ManagedCloudSdk installer = ManagedCloudSdk.newManagedSdk();
+          checkInstallation(sdkManager, installer, monitor);
+        } catch (UnsupportedOsException ex) {
+          logger.log(Level.FINE, "Unable to check Cloud SDK installation", ex);
+        } catch (ManagedSdkVerificationException ex) {
+          logger.log(Level.SEVERE, "Unable to check Cloud SDK installation. Possible causes include"
+              + " network connection problem or corrupt Cloud SDK installation.", ex);
+        } catch (ManagedSdkVersionMismatchException ex) {
+          throw new IllegalStateException("This is never thrown because we always use LATEST.", ex);
+        }
+        return Status.OK_STATUS;
+      }
+    };
+
     // Use a WorkbenchJob to trigger the check to ensure we start after the workbench window has
     // appeared, but perform the actual check within a normal Job so that we don't monopolize the
     // display thread.
-    Job triggerInstallationJob =
-        new WorkbenchJob("Check Google Cloud SDK") {
-          @Override
-          public IStatus runInUIThread(IProgressMonitor monitor) {
-            Job checkInstallationJob =
-                new Job("Check Google Cloud SDK") {
-                  @Override
-                  protected IStatus run(IProgressMonitor monitor) {
-                    try {
-                      CloudSdkManager sdkManager = CloudSdkManager.getInstance();
-                      ManagedCloudSdk installer = ManagedCloudSdk.newManagedSdk();
-                      checkInstallation(sdkManager, installer, monitor);
-                    } catch (UnsupportedOsException ex) {
-                      logger.log(Level.FINE, "Unable to check Cloud SDK installation", ex);
-                    } catch (ManagedSdkVerificationException
-                        | ManagedSdkVersionMismatchException ex) {
-                      logger.log(Level.SEVERE, "Corrupt Cloud SDK installation?", ex);
-                    }
-                    return Status.OK_STATUS;
-                  }
-                };
-            checkInstallationJob.setSystem(true);
-            checkInstallationJob.schedule(); // immediately
-            return Status.OK_STATUS;
-          }
-        };
+    Job triggerInstallationJob = new WorkbenchJob("Check Google Cloud SDK") {
+      @Override
+      public IStatus runInUIThread(IProgressMonitor monitor) {
+        checkInstallationJob.setSystem(true);
+        checkInstallationJob.schedule(); // immediately
+        return Status.OK_STATUS;
+      }
+    };
     triggerInstallationJob.setSystem(true);
     triggerInstallationJob.schedule(2000 /*ms*/);
   }

